unit1/lesson15.rst

```diff
@@ -0,0 +1,46 @@
+Unit 1 - Lesson 14 Learning Objectives
+======================================
+
+Goals
+-----
+
+- Understand how the HTTP protocol enables web communication
+- Use forms to get user data
+- Apply knowledge of HTTP to create effective form-based code
+- Use best practices to create secure, user-friendly forms
+
+Objectives
+----------
+
+HTTP
+^^^^
+
+- Identify the main parts of a URL and describe the role of each: protocol, host, port, path, query parameter, fragment
+- Understand the request/response flow of HTTP
+- Explain the two main sections of an HTTP request: headers and body
+- Name the default ports for HTTP requests
+- Name the two most common HTTP request types---``GET`` and ``POST``---and describe common use cases for each
+- Describe usage of HTTP request headers
+- Describe common request headers: ``Host``, ``User-Agent``, ``Accept``
+- Describe common HTTP response status codes: 200, 404, 500
+- Describe meaning of HTTP response headers
+- Describe common response headers: ``content-length``, ``content-type``
+- Describe the role of web servers in providing web content
+- Explain what a web application does, and how it differs from a web server
+- Identify and explain the basic role of the components of network communication: HTTP, TCP/IP, DNS
+- Explain the flow of an HTTP request made from a web browser
+- Exlain how resource loading occurs in web pages (scripts, images, etc)
+- View requests and responses using web browser dev tools 
+
+Forms
+^^^^^
+
+- Create a form using HTML
+- Understand the main form element attributes, ``action`` and ``method``
+- Properly use common inputs: text, submit, radio, checkbox, select, textarea, password, email 
+- Use label elements to add human-readable labels to form inputs
+- Use the ``name`` attribute to label form inputs
+- Explain how the ``name`` of an input impacts how data is submitted
```

Comment:
Replace the two ``name`` attribute statements with one. "-Use the ``name`` attribute to identify inputs in the submitted data"